packages/iov-tendermint-rpc/src/client.spec.ts

```diff
@@ -426,6 +427,39 @@ function websocketTestSuite(rpcFactory: () => RpcClient, adaptor: Adaptor): void
     })().catch(done.fail);
   });
 
+  it("can unsubscribe and re-subscribe to the same stream", async () => {
+    pendingWithoutTendermint();
+
```

Comment:
Yes, this is cleaner, but I don't see the unsubscribe there

Reply:
`toListPromise` unsubscribes after it got the number of elements in the second argument

Reply:
Ohhh... very nice